spec/lib/reports/subscriber_lists_report_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe Reports::SubscriberListsReport do
+  let(:created_at) { Time.zone.parse("2020-06-15").midday }
+  let(:list) { create(:subscriber_list, created_at: created_at, title: "list 1", slug: "list-1") }
```

Comment:
Minor: looks like these aren't needed outside of the `before` block.

Reply:
Good spot, this was left over from some refactoring.